sst/mtm1m3/Identify_Oscillations_Nightly.ipynb

```diff
@@ -0,0 +1,754 @@
+{
```

Comment:
<p>This failed as well. Here is the truncated traceback:</p>

<pre class="ql-syntax" spellcheck="false">`Cell In[5], line 34, in M1M3IdentifyOscillations.run(self)
     30 peaks_df_list = []
     31 for slew in self.slews:
     32     query_result = M1M3EFDQuery(
     33         slew, self.eventMaker.client, outer_pad=0
---&gt; 34     ).query_dataset()
</pre><p> </p>

<pre class="ql-syntax" spellcheck="false">TypeError: getEfdData() got an unexpected keyword argument 'raiseIfTopicNotInSchema'
</pre><p> </p><p>Can you have a look?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>is your summit_utils out of to date or on a branch that is not main or sitcom-performance-analysis?</p><p><a href="https://github.com/lsst-sitcom/summit_utils/blob/2cf903f824290b8ca1bc58749dc4a2090133fe23/python/lsst/summit/utils/efdUtils.py#L169-L184" target="_blank">https://github.com/lsst-sitcom/summit_utils/blob/2cf903f824290b8ca1bc58749dc4a2090133fe23/python/lsst/summit/utils/efdUtils.py#L169-L184</a></p>

Reply:
<p>Indeed. It was out to date. I updated it, ran again and it worked fine. </p>